extensions/default/src/getSopClassHandlerModule.js

```diff
@@ -20,9 +20,9 @@ const makeDisplaySet = instances => {
     SeriesTime: instance.SeriesTime,
     SeriesInstanceUID: instance.SeriesInstanceUID,
     StudyInstanceUID: instance.StudyInstanceUID,
-    SeriesNumber: instance.SeriesNumber,
+    SeriesNumber: instance.SeriesNumber || 0,
```

Comment:
can't it just be left undefined?

Reply:
No, undefined throws an exception if one accesses something by position.